$NetBSD: patch-aa,v 1.7 2021/07/19 13:03:39 tnn Exp $

Don't error out (skipping utmp logging) when using Unix98 ptys.

--- src/logging.c.orig	2008-02-17 22:04:50.000000000 +0000
+++ src/logging.c
@@ -105,7 +105,7 @@ rxvt_makeutent(rxvt_t *r, int page, cons
     else if (sscanf(pty, "pts/%d", &i) == 1)
 	sprintf(ut_id, "vt%02x", (i & 0xff));	/* sysv naming */
 #endif
-    else if (STRNCMP(pty, "pty", 3) && STRNCMP(pty, "tty", 3)) {
+    else if (STRNCMP(pty, "pty", 3) && STRNCMP(pty, "tty", 3) && STRNCMP(pty, "pts", 3)) {
 	rxvt_msg (DBG_ERROR, DBG_LOGGING, "can't parse tty name \"%s\"", pty);
 	return;
     }
